apps/frontend/src/app/poll/choose-events/choose-events.component.ts

```diff
@@ -52,6 +54,7 @@ export class ChooseEventsComponent implements OnInit {
   ) {
     this.mail = mailService.getMail()
     this.token = tokenService.getToken();
+    this.newParticipant.token = this.token;
```

Comment:
If you move the stuff down to `resetNewParticipant()` you could call this function here instead and move `this.token` inside that function.